source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,23 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header][] is being forwarded otherwise the basic auth to the metrics endpoint will fail.
+  - `cf update-service <cdn-route-service> \
+    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization"]}'`
 
 Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana](https://grafana-paas.cloudapps.digital).
 
+### Notes about the app route
+
+* Only the first route of the app will be bound, any changes to the first route will be picked up within 5 minutes.
```

Comment:
can we say "scraped" not "bound" here? I feel this is closer to the language that users would understand and would use themselves

Reply:
Also, this is two separate sentences separated with a comma - can you use a full stop

Reply:
Sure